### PR TITLE
Characters

### PR DESCRIPTION
**Pull request by Raman-Piachko**

### CODE REVIEW

File: src/com/epam/task/characters/Task1.java [hunk 1]
```diff
@@ -0,0 +1,91 @@
+package com.epam.task.characters;
+
+import java.util.Arrays;
+
+public class Task1 {
+    public static void main(String[] args) {
+        String[] string = {"charArrayAfsTfsfsY", "safAsffF"};
```
Comment (RamanShyrski) [inline src/com/epam/task/characters/Task1.java]: I would rename to stringArray

File: src/com/epam/task/characters/Task1.java [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+        for (char aChar : chars) {
+            if (Character.isUpperCase(aChar)) {
+                count++;
+            }
+        }
+
+        return count;
+    }
+
+    private static int[] findIndex(char[] chars) {
```
Comment (RamanShyrski) [inline src/com/epam/task/characters/Task1.java]: Rename to findUpperCaseIndexes

File: src/com/epam/task/characters/Task1.java [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+    }
+
+
+    private static void formatToLowerCase(char[] chars) {
+        for (int i = 0; i < chars.length; i++) {
+            chars[i] = Character.toLowerCase(chars[i]);
+        }
+    }
+}
+
```
Comment (RamanShyrski) [inline src/com/epam/task/characters/Task1.java]: Remove redundant lines in the end of the file

File: src/com/epam/task/characters/Task1.java [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+            } else {
+                charsAfter[i] = chars[step];
+                step++;
+            }
+        }
+
+        return charsAfter;
+    }
+
+    private static int countIndex(char[] chars) {
```
Comment (RamanShyrski) [inline src/com/epam/task/characters/Task1.java]: countUpperCaseIndexes?

File: src/com/epam/task/characters/Task2.java [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+
+    public static int[] findIndexWord(char[] chars) {
+        int[] index = new int[countIndexWord(chars)];
+        int step = 0;
+
+        for (int i = 0; i < chars.length - 3; i++) {
+            if (chars[i] == 'w' && chars[i + 1] == 'o' && chars[i + 2] == 'r' && chars[i + 3] == 'd') {
+                index[step] = i;
+                step++;
+
```
Comment (RamanShyrski) [inline src/com/epam/task/characters/Task2.java]: remove empty line-it's redundant

File: src/com/epam/task/characters/Task3.java [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+        String string = "afas fasfaf97fas as5f5a 58fas";
+        int countOfDigitsInString = countDigitsInString(string);
+        System.out.println(countOfDigitsInString);
+    }
+
+    public static int countDigitsInString(String string) {
+        char[] chars = string.toCharArray();
+        int count = 0;
+
+        for (char aChar : chars) {
```
Comment (RamanShyrski) [inline src/com/epam/task/characters/Task3.java]: aChar? I think just char will be fine

File: src/com/epam/task/characters/Task4.java [hunk 7]
```diff
@@ -0,0 +1,30 @@
+package com.epam.task.characters;
+
+public class Task4 {
+    public static void main(String[] args) {
+        String string = "214 af8 faf6886s 90 gd 2";
+        int result = countNumbersInString(string);
+        System.out.println();
+        System.out.println(result);
```
Comment (RamanShyrski) [inline src/com/epam/task/characters/Task4.java]: remove 7 line and change 8 to System.out.println("\n" + result);

**Review by RamanShyrski — CHANGES_REQUESTED** [review]

File: src/com/epam/task/characters/Task3.java (hunk 6 above)
Comment (Raman-Piachko) [reply src/com/epam/task/characters/Task3.java]: "char" is reserved word((

Comment (RamanShyrski) [reply src/com/epam/task/characters/Task3.java]: aChar is still not renamed:)

**Review by RamanShyrski — APPROVED** [review]

**Pull request merged**